docs/config/index.md

```diff
@@ -436,6 +436,22 @@ export default async ({ command, mode }) => {
 
   Restrict files that could be served via `/@fs/`. Accessing files outside this directory will result in a 403.
   
+  Because of [a bug](https://github.com/vitejs/vite/issues/3347) in vite, accessing any symbolic link while serving your project will currently fail.
```

Comment:
We should reformulate this sentence and not write `because of a bug in Vite`  
This can be rewritten in a bit more positive way somehow :)

Reply:
Maintainers have edit rights on this PR, feel free to update it to make it sound as if it were not a bug :stuck_out_tongue:

Reply:
When I think further about this, someone who wants to fix it should work on a PR and we should not add extra documentation for this, due to it could go under and then document something that is already fixed (in the future)

@antfu What do you think?

Reply:
The currently deployed version of vite links to this part of the documentation in its error message (modulo [a typo in the link](https://github.com/vitejs/vite/pull/3345)). So even when this is fixed, it will probably take some time before everyone updates to the latest version, and in the meantime, I think it's good to have a clear explanation of the problem when you click on the link in the error.